crates/db/store-sled/src/unified_mmr/schemas.rs

```diff
@@ -0,0 +1,26 @@
+use strata_db_types::mmr_helpers::{MmrId, MmrMetadata};
+use strata_primitives::buf::Buf32;
+
+use crate::{define_table_with_seek_key_codec, define_table_without_codec, impl_borsh_value_codec};
+
+define_table_with_seek_key_codec!(
+    /// Unified MMR node storage: (mmr_id, position) -> hash
+    (UnifiedMmrNodeSchema) (MmrId, u64) => Buf32
```

Comment:
this is the reverse of `UnifiedMmrHashIndexSchema`, right? Why not calling it `UnifiedMmrIndexHashSchema` for consistent language?

Also, _node_ usually refers to a tree internal node, while here I think we mean (only) leaves.